### PR TITLE
docs(install/kubernetes): Point to Configure Kubernetes from Install on Kubernetes

### PR DESCRIPTION
**Pull request by sklirg**
Suggest "Next Steps" to point to Kubernetes, which is what this page is about, and not Linux.

#### PR Checklist

- [ ] CHANGELOG updated
- [x] Documentation added
- [ ] Tests updated

### CODE REVIEW

**Comment by sklirg** [comment]
Do you want a CHANGELOG entry for updating these docs? :-)

File: docs/sources/flow/setup/install/kubernetes.md [hunk 1]
```diff
@@ -57,4 +57,4 @@ For more information on the Grafana Agent Helm chart, refer to the Helm chart do
 ## Next steps
 
 - [Start Grafana Agent]({{< relref "../start-agent#linux" >}})
```
Comment (clayton-cornell) [inline docs/sources/flow/setup/install/kubernetes.md:59]:
```suggestion
- [Start Grafana Agent]({{< relref "../start-agent" >}})
```

**Review by clayton-cornell — APPROVED** [review]
Looks good

**Comment by clayton-cornell** [comment]
Fixed broken backports to v0.33 and v0.35.

**Pull request merged**